framework/src/Volo.Abp.EntityFrameworkCore/Volo/Abp/ObjectExtending/EfCoreObjectExtensionManagerExtensions.cs

```diff
@@ -113,7 +150,7 @@ public static class EfCoreObjectExtensionManagerExtensions
             );
         }
 
-        public static void ConfigureEfCoreEntity(
+        public static void ConfigureEfCoreEntityProperties(
```

Comment:
Renaming this method (and replacing with another one which does a different thing) is a hidden breaking change which is very dangerous.
I suggest to merge these two methods (`ConfigureEfCoreEntityProperties` and `ConfigureEfCoreEntity`) that performs both entity and property mappings in a single method. This won't be a breaking change and will be more meaningful.